bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -54,7 +54,8 @@
       Status.Code.DEADLINE_EXCEEDED,
       Status.Code.INTERNAL,
       Status.Code.UNAVAILABLE,
-      Status.Code.ABORTED);
+      Status.Code.ABORTED,
```

Comment:
I'm a bit concerned about having 2 levels of retries. Won't that cause excessive retries? Auth will retry x times and the retry listener will retry y times ending up with x * y times?